rubicon_ml/client/mixin.py

```diff
@@ -410,15 +424,18 @@ class TagMixin:
     """Adds tag support to a client object."""
 
     def _get_taggable_identifiers(self):
-        dataframe_id = None
+        entity_id = None
 
-        if isinstance(self, client.Dataframe):
-            project_name, experiment_id = self.parent._get_parent_identifiers()
-            dataframe_id = self.id
-        else:
+        if isinstance(self, client.Project):
             project_name, experiment_id = self._get_parent_identifiers()
+        elif isinstance(self, client.Experiment):
```

Comment:
whats this case for?

Reply:
do we need to grab entity id for experiment as well?

Reply:
add a comment explaining what is being accomplished

Reply:
addressed in comment below